File: flight_checkin_links/v2/get/iOS SDK/FlightCheckinLinks.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.referenceData.urls.checkinLinks.get(params: ["airlineCode": "BA"],
                                            onCompletion: { result in
                                                switch result {
                                                case let .success(response):
                                                    print(response.data)
                                                case let .failure(error):
                                                    fatalError(error.localizedDescription)
                                                }
                                            })

File: flight_checkin_links/v2/get/Node SDK/flight_checkin_links.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What is the URL to my online check-in?
    const response = await amadeus.referenceData.urls.checkinLinks.get({
      airlineCode: "BA",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
